.circleci/config.yml

```diff
@@ -232,19 +232,6 @@ jobs:
               --entrypoint="pytest" poldracklab/fmriprep:latest \
               /src/fmriprep/fmriprep -svx --doctest-modules
 
-      - run:
-          name: Test fmriprep-wrapper (Python 2)
-          command: |
-            export PY2=$(pyenv versions | grep '2\.' |
-                         sed -e 's/.* 2\./2./' -e 's/ .*//')
-            pyenv local $PY2
-            echo -n "Python version: "
-            python --version
-            pip install --upgrade pip setuptools
-            pip install --upgrade wrapper/
-            which fmriprep-docker
-            fmriprep-docker -i poldracklab/fmriprep:latest --help
-            fmriprep-docker -i poldracklab/fmriprep:latest --version
```

Comment:
This is intentionally left available in Python 2 because Python 2 is still the default Python on a lot of systems, and so it's the cheapest way to get people going in Docker.